### ISSUE TITLE
Lil' Spewer execution

### ISSUE THREAD

**Issue by Lonrite** [UnderscoreKilburn/repentance-issues#2001]
kinda makes no sense from the design perspective

for example, the yellow puddle has basically non-existing range and takes a while to be fully set-up, so it needs something to make up for it right? well it should, but what actually happens instead is some trivial damage with creep that doesn't even last that long

red version of it is superior in basically everything, and the downsides it has pale in comparison to everything else. insane damage output, way better range and much faster to set up than the yellow one.

Basically my point is that there's no reason to swap this guy from his red version once it's settled, maybe make it so he gets hungry once long enough time has passed and becomes less useful? this way you'd still have reasons to take pills in a situation you normally wouldn't — no one wants an underperforming version of a familiar when you can have a different one at will
